src/target/riscv/debug_defines.c

```diff
@@ -0,0 +1,4145 @@
+/*
+ * This file is auto-generated by running 'make debug_defines' in
+ * https://github.com/riscv/riscv-debug-spec/ (d749752-dirty)
```

Comment:
This is intentional. I would like to avoid updating the version of the spec used to generate this file, since it would require a lot of changes in code (some constants were renamed and so on). The version is marked as dirty to indicate that the generators were cherry-picked on top of `d749752`. If you insist, I will update `debug_defines.h` to current version, though I'm not sure if it is the best time to do so.

Reply:
> I would like to avoid updating the version of the spec used to generate this file, since it would require a lot of changes in code (some constants were renamed and so on). 

Understood. In that case, it is my recommendation to leave this merge request as is, but please make a proper upgrade of the debug_defines (from a clean commit) in a subsequent merge request.

> The version is marked as dirty to indicate that the generators were cherry-picked on top of d749752

This is exactly the reason why I suggest to use a clean commit -- we can't expect other developers to figure out what needs to be cherry-picked where in order to replicate your work (e.g. during troubleshooting).

Thanks.